src/Datalake2Anomali.py

```diff
@@ -240,14 +425,56 @@ def _generateIndicators(self, bulk_searches_results, valid_datalake_queries):
         return indicators
 
     def uploadIndicatorsToAnomali(self):
-        anomali_api = AnomaliApi(
-            ssl_verify=config.ssl_verify, proxies=config.proxies, logger=self.logger
-        )
         bulk_searches_results, valid_datalake_queries = self._getDatalakeThreats()
         indicators = self._generateIndicators(
             bulk_searches_results, valid_datalake_queries
         )
-        payload = anomali_api._prepareIndicatorPayload(indicators=indicators)
-        anomali_api.uploadPayload(payload)
+        payload = self.anomali_api._prepareIndicatorPayload(indicators=indicators)
+        self.anomali_api.uploadPayload(payload)
 
         return
+
+    def _get_bulletins_from_world_watch(self):
+        updated_after_string = (datetime.now() - timedelta(hours=config.upload_frequency)).strftime(WORLD_WATCH_TIME_FORMAT)
+
+        if not config.run_as_cron:
+            last_time = self.anomali_api.get_datetime_of_last_world_watch_report()
+
+            if last_time:
+                updated_after_string = last_time.strftime(WORLD_WATCH_TIME_FORMAT)
+
+        advisory_endpoint = f"{self.world_watch_url}/api/advisory/"
+
+
+        r = requests.get(advisory_endpoint,
+            headers=self.world_watch_headers,
+            params={
+                "updated_after": updated_after_string
+        })
+
+        if r.status_code not in SUCCESSFUL_HTTP_CODES:
+            raise HTTPException(f"Cannot get bulletins from world watch, {r.content}, {r.status_code}")
+
+        specific_advisory_endpoint = f"{self.world_watch_url}/api/advisory"
+        complete_advisories = []
+
+        for item in r.json()['items']:
```

Comment:
You can use instead "html" endpoint to get the full latest version of the report in HTML format.
```suggestion
        for item in r.json()['items']:
            r = requests.get(f"{specific_advisory_endpoint}/{item['id']}/html", headers=self.world_watch_headers)

            if r.status_code not in SUCCESSFUL_HTTP_CODES:
                raise HTTPException(f"Cannot get complete advisory, {r.content}, {r.status_code}")

            complete_advisory = item
            complete_advisory['html'] = r.json()['html']
            complete_advisories.append(complete_advisory)
```